libpod/container_internal.go

```diff
@@ -1112,6 +1115,16 @@ func (c *Container) stop(timeout uint) error {
 		return err
 	}
 
+	// Old versions of conmon have a bug where they create the exit file before
+	// closing open file descriptors causing a race condition when restarting
+	// containers with open ports since we cannot bind the ports as they're not
+	// yet closed by conmon.
+	//
+	// Killing the old conmon PID is ~okay since it forces the FDs of old conmons
+	// to be closed, while it's a NOP for newer versions which should have
+	// exited already.
+	unix.Kill(conmonPID, unix.SIGKILL)
```

Comment:
Should make this conditional on conmonPID != 0 - older Podman versions may not have it properly set

Reply:
Just updated. Thanks!